# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/linux25/06-devfs-default.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./arch/ppc/defconfig.orig	2003-01-13 19:17:42.000000000 +0100
+++ ./arch/ppc/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -728,10 +728,6 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-CONFIG_DEVFS_FS=y
-# CONFIG_DEVFS_MOUNT is not set
-# CONFIG_DEVFS_DEBUG is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_QNX4FS_RW is not set
 # CONFIG_ROMFS_FS is not set
--- ./arch/i386/defconfig.orig	2003-01-13 19:17:38.000000000 +0100
+++ ./arch/i386/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -738,8 +738,6 @@
 # CONFIG_NTFS_FS is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_ROMFS_FS is not set
 CONFIG_EXT2_FS=y
--- ./arch/sparc/defconfig.orig	2003-01-13 19:17:40.000000000 +0100
+++ ./arch/sparc/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -294,8 +294,6 @@
 # CONFIG_NTFS_FS is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 CONFIG_ROMFS_FS=m
 CONFIG_EXT2_FS=y
--- ./arch/m68k/defconfig.orig	2003-01-13 19:17:10.000000000 +0100
+++ ./arch/m68k/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -231,10 +231,6 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
-# CONFIG_DEVFS_DEBUG is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_ROMFS_FS is not set
 CONFIG_EXT2_FS=y
--- ./arch/sparc64/defconfig.orig	2003-01-13 19:17:19.000000000 +0100
+++ ./arch/sparc64/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -615,8 +615,6 @@
 # CONFIG_NTFS_FS is not set
 CONFIG_HPFS_FS=m
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 CONFIG_ROMFS_FS=m
 CONFIG_EXT2_FS=y
--- ./arch/m68knommu/defconfig.orig	2003-01-13 19:17:39.000000000 +0100
+++ ./arch/m68knommu/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -449,10 +449,6 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
-# CONFIG_DEVFS_DEBUG is not set
-# CONFIG_DEVPTS_FS is not set
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_QNX4FS_RW is not set
 CONFIG_ROMFS_FS=y
--- ./arch/alpha/defconfig.orig	2003-01-13 19:17:34.000000000 +0100
+++ ./arch/alpha/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -603,8 +603,6 @@
 # CONFIG_NTFS_FS is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_ROMFS_FS is not set
 CONFIG_EXT2_FS=y
--- ./arch/cris/defconfig.orig	2003-01-13 19:17:16.000000000 +0100
+++ ./arch/cris/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -444,10 +444,6 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
-# CONFIG_DEVFS_DEBUG is not set
-# CONFIG_DEVPTS_FS is not set
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_QNX4FS_RW is not set
 # CONFIG_ROMFS_FS is not set
--- ./arch/mips/defconfig.orig	2003-01-13 19:17:11.000000000 +0100
+++ ./arch/mips/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -446,10 +446,6 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
-# CONFIG_DEVFS_DEBUG is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_QNX4FS_RW is not set
 # CONFIG_ROMFS_FS is not set
--- ./arch/x86_64/defconfig.orig	2003-01-13 19:17:38.000000000 +0100
+++ ./arch/x86_64/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -517,8 +517,6 @@
 # CONFIG_NTFS_FS is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_ROMFS_FS is not set
 CONFIG_EXT2_FS=y
--- ./arch/ppc64/defconfig.orig	2003-01-13 19:17:15.000000000 +0100
+++ ./arch/ppc64/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -489,8 +489,6 @@
 # CONFIG_NTFS_FS is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_ROMFS_FS is not set
 CONFIG_EXT2_FS=y
--- ./arch/um/defconfig.orig	2003-01-13 19:17:37.000000000 +0100
+++ ./arch/um/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -195,10 +195,6 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-CONFIG_DEVFS_FS=y
-CONFIG_DEVFS_MOUNT=y
-# CONFIG_DEVFS_DEBUG is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_QNX4FS_RW is not set
 # CONFIG_ROMFS_FS is not set
--- ./arch/arm/defconfig.orig	2003-01-13 19:17:16.000000000 +0100
+++ ./arch/arm/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -424,10 +424,6 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
-# CONFIG_DEVFS_DEBUG is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_QNX4FS_RW is not set
 CONFIG_ROMFS_FS=y
--- ./arch/parisc/defconfig.orig	2003-01-13 19:17:10.000000000 +0100
+++ ./arch/parisc/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -566,10 +566,6 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
-# CONFIG_DEVFS_DEBUG is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_QNX4FS_RW is not set
 # CONFIG_ROMFS_FS is not set
--- ./arch/ia64/defconfig.orig	2003-01-13 19:17:11.000000000 +0100
+++ ./arch/ia64/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -547,8 +547,6 @@
 # CONFIG_NTFS_FS is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_ROMFS_FS is not set
 CONFIG_EXT2_FS=y
--- ./arch/mips64/defconfig.orig	2003-01-13 19:17:41.000000000 +0100
+++ ./arch/mips64/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -404,10 +404,6 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
-# CONFIG_DEVFS_DEBUG is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_QNX4FS_RW is not set
 # CONFIG_ROMFS_FS is not set
--- ./arch/s390x/defconfig.orig	2003-01-13 19:17:07.000000000 +0100
+++ ./arch/s390x/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -307,8 +307,6 @@
 # CONFIG_NTFS_FS is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_ROMFS_FS is not set
 CONFIG_EXT2_FS=y
--- ./arch/sh/defconfig.orig	2003-01-13 19:17:35.000000000 +0100
+++ ./arch/sh/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -159,10 +159,6 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
-# CONFIG_DEVFS_DEBUG is not set
-# CONFIG_DEVPTS_FS is not set
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_QNX4FS_RW is not set
 # CONFIG_ROMFS_FS is not set
--- ./arch/s390/defconfig.orig	2003-01-13 19:17:42.000000000 +0100
+++ ./arch/s390/defconfig	2003-01-14 14:49:32.000000000 +0100
@@ -246,8 +246,6 @@
 # CONFIG_NTFS_FS is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
 # CONFIG_ROMFS_FS is not set
 CONFIG_EXT2_FS=y
